test/integration/noderesourcetopology_test.go

```diff
@@ -215,7 +214,7 @@ func TestTopologyMatchPlugin(t *testing.T) {
 		{
 			name: "Filtering out nodes that cannot fit resources on a single numa node in case of Guaranteed pod",
 			pods: []*v1.Pod{
-				withContainer(withReqAndLimit(st.MakePod().Namespace(ns.Name).Name("topology-aware-scheduler-pod"), map[v1.ResourceName]string{v1.ResourceCPU: "4", v1.ResourceMemory: "50Gi"}).Obj(), pause),
+				withContainer(withReqAndLimit(st.MakePod().Namespace(ns.Name).Name("topology-aware-scheduler-pod"), map[v1.ResourceName]string{v1.ResourceCPU: "4", v1.ResourceMemory: "5Gi"}).Obj(), pause),
```

Comment:
Is it intentional to change to 5Gi?

Reply:
yes, it was intentional. Now we don't take into account memory amount, since we didn't know about memory NUMA id assigned.
But this is for further changes (maybe a separate commit), where we will take into account memory amount, since MemoryManager now supports podresources.